rtloader/common/stringutils.c

```diff
@@ -44,10 +45,12 @@ char *as_string(PyObject *object)
         PyErr_Clear();
         return NULL;
     }
-    retval = strdupe(tmp);
-#else
-    PyObject *temp_bytes = NULL;
 
+    temp_bytes = object;
+    retval = tmp;
+    Py_INCREF(temp_bytes);
+    *stringObject = temp_bytes;
```

Comment:
`temp_bytes` isn't needed here.
```suggestion
    retval = tmp;
    Py_INCREF(object);
    *stringObject = object;
```